sal_interfaces/SALSubsystems.xml

```diff
@@ -556,12 +556,12 @@
 <SALSubsystem>
   <Name>EAS</Name>
   <Description>The CSC for the Environmental Awareness System</Description>
-  <IndexEnumeration>first,second</IndexEnumeration>
+  <IndexEnumeration>no</IndexEnumeration>
   <Generics>yes</Generics>
   <ActiveDevelopers>Wouter van Reeven</ActiveDevelopers>
   <Github></Github>
```

Comment:
Isn't there a GH URL for this already?

Reply:
It turns out there is and I wasn't aware of that.